gdk-pixbuf build error: "ImportError: No module named site"

### Steps to reproduce the issue

```console
$ spack install gdk-pixbuf
...
5 errors found in build log:
     851     /usr/bin/install -c -m 644 ./html/rn02.html
     852     /usr/bin/install -c -m 644 ./html/style.css
     853     /usr/bin/install -c -m 644 ./html/up-insensitive.png
     854     /usr/bin/install -c -m 644 ./html/up.png
     855    ImportError: No module named site
     856    Makefile:908: recipe for target 'install-data-local' failed
  >> 857    make[4]: *** [install-data-local] Error 1
     858    make[4]: Leaving directory '/tmp/user/spack-stage/spack-stage-zy7tzs/gdk-pixbuf-
            2.31.2/docs/reference/gdk-pixbuf'
...
```

### Platform and user environment

Please report your OS here:
```commandline
$ uname -a
Linux hostx 4.15.0-38-generic #41-Ubuntu SMP Wed Oct 10 10:59:38 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
$ lsb_release -d
Description:    Linux Mint 19 Tara
``` 

### Additional information
Working with the latest pull of the develop branch
[spack-build.out.gdk-pixbuf.txt](https://github.com/spack/spack/files/2596212/spack-build.out.gdk-pixbuf.txt)

--- a/var/spack/repos/builtin/packages/gdk-pixbuf/package.py
+++ b/var/spack/repos/builtin/packages/gdk-pixbuf/package.py
@@ -32,3 +32,14 @@ class GdkPixbuf(AutotoolsPackage):
                                self.prefix.share)
         run_env.prepend_path("XDG_DATA_DIRS",
                              self.prefix.share)
+
+    def configure_args(self):
+        args = []
+        # disable building of gtk-doc files following #9771
+        args.append('--disable-gtk-doc-html')
+        true = which('true')
+        args.append('GTKDOC_CHECK={0}'.format(true))
+        args.append('GTKDOC_CHECK_PATH={0}'.format(true))
+        args.append('GTKDOC_MKPDF={0}'.format(true))
+        args.append('GTKDOC_REBASE={0}'.format(true))
+        return args

